flask/blueprints.py

```diff
@@ -105,6 +106,19 @@ def __init__(self, name, import_name, static_folder=None,
         if url_defaults is None:
             url_defaults = {}
         self.url_values_defaults = url_defaults
+        if cli_group_name is None:
+            cli_group_name = name
+        self.cli_group_name = cli_group_name
+        if cli_group_desc is None:
+            cli_group_desc = 'Commands provided by {0}.'.format(self.name)
```

Comment:
good idea; that way only the "blueprint" is truncated if the name is too long, and at least with my test blueprint ("yellowprint") nothing was truncated at all